data/filters/DataWarrior/README.md

```diff
@@ -0,0 +1,76 @@
+# Use DataWarrior to create Enamine Building Block file
+
+## Table of contents
+1. Open new notebook in notebooks/custom-kinfraglib folder
+2. Load fragment library
+3. save fragment library for DataWarrior
```

Comment:
I would summarize these three steps under the headline "1. Prepare KinFragLib fragment library for DataWarrior"

Reply:
Good idea :)